### ISSUE TITLE
Review the importance of setting the sufficient access modes

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#2679]
## Issue on the [“EPUB Accessibility 1.1”](https://www.w3.org/TR/epub-a11y-11/) Recommendation and the [“EPUB Accessibility Techniques 1.1”](https://www.w3.org/TR/epub-a11y-tech-11/) WG Note

- https://www.w3.org/TR/epub-a11y-11/#sec-disc-package
- https://www.w3.org/TR/epub-a11y-tech-11/#accessModeSufficient

## Describe the problem 

Setting accessModeSufficient is not required to conform to the EPUB Accessibility specification but it's very helpful to know in the context of display metadata.

Some options to fix this include:

- change the conformance level to a requirement as part of the next conformance identifier, but this isn't a great option as it will make existing content not forward compatible
- see if there is a way in the techniques to highlight the importance of setting sufficient access modes, although the text introduced in the last revision around the importance of single sufficient access modes may already have this covered.

**Comment by clapierre** [comment]
In Benetech's Global Certified Accessible program we have since the Accessibility 1.0 has always required accessModeSufficient be included in the metadata.

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 01 May 2025](https://w3c.github.io/pm-wg/minutes/2025-05-01-a11y.html#b66d).

<details><summary><i>View the transcript</i></summary>

<h3>AccessmodeSufficient is not compulsory</h3>
<p><cite>&lt;AvneeshSingh&gt;</cite> <a href="https://github.com/w3c/epub-specs/issues/2679" rel="noopener noreferrer">w3c/<wbr>epub-specs#2679</a></p>
<p><cite>AvneeshSingh:</cite> user experience guide is very dependent on it.</p>
<p><cite>CharlesL:</cite> GCA requires it.</p>
<p><cite>mgarrish:</cite> it can be repetitive metadata.</p>
<p><cite>gpellegrino:</cite> end user it is more important to know AccessModeSufficient (AMS) instead of just AccessMode (AM). I think it should be required and AM could be optional.</p>
<p><cite>CharlesL:</cite> that could be a problem for backward compatibility.</p>
<p><cite>mgarrish:</cite> ISO specs to be backward compatibility is a concern.</p>
<p><cite>gpellegrino:</cite> making AMS required is important. mapping AM is far less obvious. lets focus on AMS.<br>
<span>… we could wait until the next major version.  metadata by LIA / GCA is already requiring it. l</span></p>
<p><cite>AvneeshSingh:</cite> We can have informative text if you really want users to discover non visual reading you need to put this "textual" in AMS.  If we can make the text stronger might be useful here.</p>
<p><cite>gpellegrino:</cite> maybe add a note.</p>
<p><cite>mgarrish:</cite> whole note is informative. we tried to add the singular AccessModes. I will look into it again.</p>
<p><cite>gpellegrino:</cite> ACE could manage this as well, minor issue to include AccessModeSufficient.</p>
<p>Charles +1<br>
<span>… are we decided it will be 1.1.1?</span></p>
<p><cite>mgarrish:</cite> : if we go to 1.2 we need to then add new conformance values.</p>
<p><cite>gpellegrino:</cite> we may jump to 1.2 if we need to elevate to WCAG 2.2 or something like Fixed Layout comes. but.</p>

<hr /></details>

**Comment by GeorgeKerscher** [comment]
If we move to version 1.2, I think accessModeSufficient should be required.

In either case ACE should flag it as being needed, if not technically required.

Would this impact ISO?

Older publications would conform to 1.0 or 1.1, but we need to move forward with improving accessibility requirements.

**Comment by gregoriopellegrino** [comment]
> If we move to version 1.2, I think accessModeSufficient should be required.

I agree

> In either case ACE should flag it as being needed, if not technically required.

I'm not sure: for me ACE should stick to the specs, so not flagging what is not mandatory.

> Older publications would conform to 1.0 or 1.1, but we need to move forward with improving accessibility requirements.

We could ask Rick, how many of the EPUBs he has have `accessMode`, but not `accessModeSufficient`, to see how impactful that would be.

**Comment by mattgarrish** [comment]
> Older publications would conform to 1.0 or 1.1

Would they? Does it lead to requiring both accessMode and accessModeSufficient or would this change also include demoting accessMode to recommended or optional?

**Comment by mattgarrish** [comment]
The decision on the task force call today was to require both accessMode and accessModeSufficient.

By only upping accessModeSufficient to a requirement and not demoting accessMode to recommended, 1.2-conformant publications will be conforming to earlier versions.

But, this will require changing the specification number to 1.2 to create a new conformance identifier -- so that existing 1.1-conformant publications without accessModeSufficient don't become non-conforming -- we're going to leave the issue open for a while for further feedback.

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 29 May 2025](https://w3c.github.io/pm-wg/minutes/2025-05-29-a11y.html#9c5b).

<details><summary><i>View the transcript</i></summary>

<h3>Decision on AccessmodeSufficient.</h3>
<p><cite>&lt;AvneeshSingh&gt;</cite> <a href="https://github.com/w3c/epub-specs/issues/2679" rel="noopener noreferrer">w3c/<wbr>epub-specs#2679</a></p><p><strong>RESOLUTION:</strong> add properties for publisher and trusted intermediary contacts</p>
<p><cite>mgarrish:</cite> it's problematic to reduce requirements in new standards as it makes it incompatible with older versions</p>
<p><cite>George:</cite> maybe we should only make access mode sufficient required</p>
<p><cite>Madeleine5:</cite> access mode sufficient is the most complicated to understand</p>
<p><cite>George:</cite> it's repeatable but we allow textual and visual as single statements</p>
<p><cite>AvneeshSingh:</cite> can this be automated</p>
<p><cite>Madeleine5:</cite> yes, I've automated these in the past to make access modes from other metadata</p>
<p><cite>AvneeshSingh:</cite> daisy pipeline computes metadata when it compiles an epub - but need to check if it works with edge cases</p>
<p><cite>AvneeshSingh:</cite> we could bump it up to mandatory and see if people resist</p>
<p><cite>Madeleine5:</cite> so we want compatibility with previous</p>
<p><cite>mgarrish:</cite> content doesn't break if we change the conformance identifier</p>
<p><cite>AvneeshSingh:</cite> let's update the tracker that we're going to move it to a requirement and if we don't get any negative feedback after a couple of weeks we can move it to the spec</p>

<hr /></details>

**Comment by mattgarrish** [comment]
A thought on this, if we're going to route of bumping the version number and adding new conformance strings, we might as well also shift accessMode to a recommended property.

It's ideal timing in that we haven't yet submitted the errata for 1.0, so we can add it to the change for summary. But we should commit to making these changes so we don't hold up the process of the errata changes.
